sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableServiceImpl.java

```diff
@@ -656,4 +697,118 @@ public int compareTo(@Nonnull EndPoint o) {
           .result();
     }
   }
+
+  static class BeamRowAdapter implements RowAdapter<com.google.bigtable.v2.Row> {
```

Comment:
BeamRow is not quite the correct name for this...Someone reading this would assume this is the Beam SQL Row, I would call this RowProtoAdapter or similar